banzai_floyds/tests/data/orders_e2e_fits.dat

```diff
@@ -0,0 +1,77 @@
+ {
```

Comment:
This is the order center polynomials as a result of doing a manual data reduction on our test data set to be used in the e2e tests for comparison.